Source/WebCore/rendering/RenderObject.h

```diff
@@ -467,7 +467,7 @@ class RenderObject : public CachedImageClient, public CanMakeCheckedPtr {
     bool isLegacySVGForeignObject() const { return type() == Type::LegacySVGForeignObject; }
     bool isSVGForeignObject() const { return type() == Type::SVGForeignObject; }
     virtual bool isLegacySVGResourceContainer() const { return false; }
-    virtual bool isSVGResourceContainer() const { return false; }
+    bool isSVGResourceContainer() const { return type() == Type::SVGResourceClipper; }
```

Comment:
I think this is now a little bit confusing. I would suggest making this function return `isSVGResourceClipper()` instead of checking then type. I would suggest also adding a FIXME comment saying more checks for resource renderers should be added once they make inherit from `RenderSVGResourceContainer` (masker, marker, pattern, filter and gradient).

Reply:
I don't see how checking for `isSVGResourceClipper()` is better than `type() == Type::SVGResourceClipper`. It would certainly not be consistent with my other checks.

With regards to the FIXME, it's not clear where I'd put it. I basically need to add an ASSERT() in each renderer constructor. I plan on following up shortly to add them.